src/app/api/endpoints/alerts.py

```diff
@@ -146,3 +147,17 @@ async def fetch_ongoing_alerts(
         )
         retrieved_alerts = [x.__dict__ for x in retrieved_alerts.all()]
         return retrieved_alerts
+
+
+@router.get("/event-alerts/{event_id}/", response_model=List[AlertOut], summary="Get the list of alerts for event")
```

Comment:
Just a suggestion:
- with this, the route will be `alerts/event-alerts/event_id`
- common public APIs would go more for the `events/event_id/alerts`

The rest is all good :+1: